java/code/src/com/redhat/rhn/manager/channel/ChannelManager.java

```diff
@@ -2427,7 +2428,7 @@ public static Long lookupOriginalId(Channel channel) {
      * @return list of errataOverview objects that need to be resynced
      */
     public static List<ErrataOverview> listErrataNeedingResync(Channel c, User user) {
```

Comment:
I think the method `removeErrata`  is missing the same fix

Reply:
That one's also fixed in `suseChannelUserRoleView` view